portfolio/src/main/webapp/style.css

```diff
@@ -1,28 +1,97 @@
+* {
+  font-family: 'Arial', 'serif';
+}
+
 body {
   margin: 0;
-  background-color: #dadada;
+  /* background-color: #99ccff; */
```

Comment:
Can this code be deleted? It is usually better to not commit commented code.

Reply:
Hi Travis,
Thank you for the feedback! I will delete the commented code and keep that in mind for the future. Looking forward to working with you this summer.
Have a great day!